app/scripts/modules/kubernetes/src/pipeline/stages/runJob/configureJob.controller.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 const angular = require('angular');
+import {KUBERNETES_LIFECYCLE_HOOK_CONFIGURER} from '../../../container/lifecycleHook.component';
```

Comment:
Spaces around the import (`{ KUBERNETES_LIFECYCLE_HOOK_CONFIGURER }`), and you should be able to make the import path `'kubernetes/container/lifecycleHook.component'`.